packages/frontend/src/components/Nav.tsx

```diff
@@ -2,24 +2,29 @@ import { Button, Drawer, IconButton, Menu, MenuItem, ButtonBase } from '@materia
 import Hidden from '@material-ui/core/Hidden'
 import { createStyles, makeStyles } from '@material-ui/core/styles'
 import Typography from '@material-ui/core/Typography'
+import { Modal, Box } from '@material-ui/core'
```

Comment:
lets use our own Modal instead of material-ui's. checkout this file:
https://github.com/opynfinance/squeeth-monorepo/blob/main/packages/frontend/src/components/StrikeWarning.tsx#L22